shell/bashrc

```diff
@@ -329,7 +329,6 @@ function phpbrew ()
             unset PHPBREW_PHP
             unset PHPBREW_PATH
             eval `$BIN env`
-            __phpbrew_set_path
```

Comment:
The `PATH` will be set at the end of `__phpbrew_reinit`. If set it right now but there's no system PHP, there won't be an interpreter available to run `phpbrew env` and it will fail with "/usr/bin/env: ‘php’: No such file or directory".